api_client/python/timesketch_api_client/client.py

```diff
@@ -605,8 +605,87 @@ def list_sigma_rules(self, as_pandas=False):
             rules.append(index_obj)
         return rules
 
+    def list_sigmarules_rules(self, as_pandas=False):
```

Comment:
Can we call this list_sigmarules instead. So many `rules` in the name otherwise.. :)

Reply:
Would it make sense to use a name that includes where the data is coming from? e.g. `list_sigma_rules_from_db`?

Reply:
done
@jkppr we could but as we do not have any other option at the moment it would just add clutter